lldb/tools/lldb-dap/Breakpoint.cpp

```diff
@@ -0,0 +1,76 @@
+//===-- Breakpoint.cpp ------------------------------------------*- C++ -*-===//
```

Comment:
Nit: you don't need the `*- C++ -*` in source files as emacs can tell it's C++ by the extension, unlike in the header.